src/main/java/uk/gov/justice/digital/delius/utils/DateConverter.java

```diff
@@ -0,0 +1,24 @@
+package uk.gov.justice.digital.delius.utils;
+
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+import java.time.LocalTime;
+import java.time.OffsetDateTime;
+import java.time.ZoneId;
+
+public class DateConverter {
+
+    private static final String LOCAL_TIMEZONE = "Europe/London";
```

Comment:
I think you could do system timezone since we are always London Europe as define in Docker file